docs/content/webhooks-app.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Webhooks App
+metaDescription: Effortlessly set up and send webhooks with our comprehensive guide.
+---
+
+# Setting Up the App Instance
+
+Begin by installing the webhooks app from the Studio App store. Once you've named your instance, you'll get a unique webhook URL and an API key, both essential for sending webhook requests.
```

Comment:
Just a suggestion to maybe add the text in bold:
Begin by installing the **W**ebhooks app from the Studio App store. Create a new instance, after which **you'll then name your webhook and click the Generate webhook button. This will return** a unique webhook URL and an API key, both essential for sending webhook requests.